_content/tour/ind/struct-types.article

```diff
@@ -138,108 +138,108 @@ fields. This causes more padding inside the struct.
+dan menginisialisasi dua variabel.
 
     var ex1 example1
     var ex2 example2
 
-You can’t assign these two variables to each other since they are of different named
-types. The fact that they are identical in structure is irrelevant. 
+Anda tidak dapat mengisi dua variabel ini satu sama lain karena mereka adalah tipe struct bernama yang berbeda.
+Kenyataan bahwa mereka identik dalam struktur tidak relevan.
 
-    ex1 = ex2  // Not allowed, compiler error
+    ex1 = ex2  // Tidak diperbolehkan, kompiler error
 
-To perform this assignment, you would have to use conversion syntax and since they
-are identical in structure, the compiler will allow this.
+Untuk melakukan pengisian ini, Anda harus menggunakan sintaks konversi dan karena mereka
+identik dalam struktur, kompiler akan mengizinkan operasi ini.
 
-    ex1 = example1(ex2)  // Allowed, NO compiler error
+    ex1 = example1(ex2)  // Diperbolehkan, tidak ada kompiler error
 
-However, if ex2 was changed to be declared as an unnamed type using the same exact
-declaration as ex1, no conversion syntax would be required.
+Namun, jika ex2 diubah untuk dideklarasikan sebagai tipe struct tanpa nama menggunakan deklarasi yang sama persis
+seperti ex1, tidak ada sintaks konversi yang diperlukan.
 
     var ex2 struct {
         flag    bool
         counter int16
         pi      float32
     }
 
-    ex1 = ex2  // Allowed, NO need for conversion syntax
+    ex1 = ex2  // Diperbolehkan, tidak perlu ada konversi
 
-The compiler will allow this assignment without the need for conversion.
+Kompiler akan mengizinkan assignment ini tanpa perlu konversi.
 
-** Notes
+** Catatan
 
-- We can use the struct literal form to initialize a value from a struct type.
-- The dot (.) operator allows us to access individual field values.
-- We can create anonymous structs.
+- Bentuk literal struct dapat digunakan untuk menginisialisasi value dari tipe struktur.
+- Operator titik (.) memungkinkan untuk mengakses value dari suatu atribut.
+- Kita dapat membuat tipe struct tanpa nama.
 
 ** Quotes
 
 "Implicit conversion of types is the Halloween special of coding. Whoever thought
 of them deserves their own special hell." - Martin Thompson
 
-** Extra Reading
+** Bacaan Ekstra
 
-- [[https://www.ardanlabs.com/blog/2013/07/understanding-type-in-go.html][Understanding Type in Go]] - William Kennedy    
-- [[https://www.ardanlabs.com/blog/2013/07/object-oriented-programming-in-go.html][Object Oriented Programming in Go]] - William Kennedy    
-- [[https://dave.cheney.net/2015/10/09/padding-is-hard][Padding is hard]] - Dave Cheney    
-- [[https://www.geeksforgeeks.org/structure-member-alignment-padding-and-data-packing/][Structure Member Alignment, Padding and Data Packing]]    
-- [[http://www.catb.org/esr/structure-packing][The Lost Art of Structure Packing]] - Eric S. Raymond    
+- [[https://www.ardanlabs.com/blog/2013/07/understanding-type-in-go.html][Memahami tipe di Go]] - William Kennedy    
```

Comment:
I prefer not to translate the tittle of those listed articles

Reply:
ack